es/es-server/src/main/java/org/elasticsearch/cluster/InternalClusterInfoService.java

```diff
@@ -396,30 +396,45 @@ static void fillDiskUsagePerNode(Logger logger,
                 }
                 String nodeId = nodeStats.getNode().getId();
                 String nodeName = nodeStats.getNode().getName();
-                var leastAvailableTotals = leastAvailablePath.getTotal();
```

Comment:
is this optimization that necessary and beneficial?

Reply:
I would go without, makes it just harder to maintain with the es code.